content/60.methods.md

```diff
@@ -99,82 +99,125 @@ For instance, the template variable {% raw %}`{{ebm_trials_results}}`{% endraw %
+Manubot uses Zotero [@url:https://www.zotero.org] to extract metadata for some types of citations, but Zotero did not support clinical trial identifiers.
+In order to enable Manubot to pull metadata associated with clinical trials based on their identifiers, we needed to develop Zotero support for these identifiers.
+The need for this feature to be enabled in Zotero had also been identified by other researchers [@url:https://forums.zotero.org/discussion/74933/import-from-clinical-trials-registry; @url:https://forums.zotero.org/discussion/77721/add-reference-from-clinical-trials-org].
+This update was achieved through the implementation of a query of clinicaltrials.gov to retrieve XML metadata associated with each identifier using JavaScript [@url:https://github.com/zotero/translators/pull/2153]. 
+
+Another challenge that emerged was that Because of the large number of citations used in this manuscript and the fast-moving nature of COVID-19 research, keeping track of retractions, corrections, and notices of concern became a priority.
+A new plugin was added to Manubot to support "smart citations" in the HTML build of manuscripts.
 The plugin uses the [Scite](https://scite.ai/) service to display a badge below any citation with a DOI.
 The badge contains a set of icons and numbers that indicate how many times that source has been mentioned, supported, or disputed, and whether there have been any important editorial notices, such as retractions or corrections.
 Using this, we were able to quickly identify references that needed to be checked again since the time they had been added.
 This was invaluable given the nature of the project, where we were disseminating rapidly evolving information of great consequence from over a thousand different sources.
 The badges also allow readers to roughly evaluate the reliability of cited sources at a glance.
 
 Because in this implementation of Manubot, most collaborators were writing and editing text through the GitHub website rather than in a local text editor, we also needed to add spell-checking functionalities to Manubot.
-<!--To Do: Describe-->
+<!--To Do: Describe -- perhaps @agitter can lead this?-->
 
+Manubot provides the advantage of allowing a manuscript to be rendered in several formats that serve different purposes, and the current project extended these options.
+Docx is a necessary format for a biological collaboration where authors are typically not working in LaTeX.
+<!--To Do: continue expanding to describe docx updates and possibly LaTeX updates-->
 <!-- To Do: Also, if we end up automating anything to do with the Manubot-to-LaTeX or bibtex workflow while trying to submit this paper, we should describe that too, as well as the udpates for the Microsoft Word crowd to generating the complete review manuscript along with the individual manuscripts reviewing specific topics. <!-- To Do: reference individual manuscripts -->
```

Comment:
@mprobson if you want to add some info about LaTeX, this would be the place! I'll work on expanding this paragraph as well about the docx stuff (as @agitter mentioned above)

Reply:
I'm writing something for this